pkg/queue/manager.go

```diff
@@ -274,6 +279,10 @@ func (m *Manager) deleteWorkloadFromQueueAndClusterQueue(w *kueue.QueuedWorkload
 	cq := m.clusterQueues[q.ClusterQueue]
 	if cq != nil {
 		cq.Delete(w)
+		if w.Spec.Admission != nil {
+			cq.MoveAllToClusterQueue()
```

Comment:
the workload could have borrowed capacity from another CQ. This MoveAll needs to happen for the cohort, not just this CQ.

Reply:
Thanks updated.